CryptoBinPkg/Driver/readme.md

```diff
@@ -1,54 +1,99 @@
-# Crypto Driver
+# Shared Crypto Binaries
 
-VERSION: 2.0
+Shared crypto consists of a set of pre-built binaries that provide cryptographic services. The services are exposed
+to other modules via dynamic interfaces such as PPIs in the PEI phase and protocols in the DXE and MM phases
 
-This is a pre-built version of BaseCryptLib and TlsLib built into binaries that make the services available to other
-modules via dynamic interfaces such as PPIs and protocols.
+**This project does not directly implement crypto algorithms**. Crypto algorithm implementation is provided by an
+underlying crypto provider. The provided may change over time based on industry shifts in crypto requirements,
+enhancements for embedded crypto support in available crypto projects, and overall crypto provider quality and
+maintainership.
 
-This currently supports Openssl implementation of BaseCryptLib but this will be expanded in the future.
+- Currently, the OpenSSL crypto project is used.
 
-## Benefits
+Any consumer of this binary should expect that the underlying crypto provider may change. Consumers are encouraged to
+participate in all aspects of shared crypto including code contributions and feedback.
+
+## Terminology
 
-But first, why would you care about this?
+- **Crypto Provider** - The project that provides an implementation of cryptographic algorithms used by this project
+  those crypto services to platform firmware.
+  - Examples: [OpenSSL](https://github.com/openssl/openssl), [Mbed TLS](https://github.com/Mbed-TLS/mbedtls),
+    [SymCrypt](https://github.com/microsoft/SymCrypt).
+- **Flavor** - A collection of cryptographic algorithms included in a given shared crypto binary.
+- **Phase** - The "boot phase" that the binary applies to where a phase is defined in the [Platform Initialization

```

Comment:
small nit: should this be Module Type instead of Phase?

Reply:
Yes, I moved the example to a Module Type section. Left the Phase definition as it was.